spec/factories/tasuku_taskables_questions.rb

```diff
@@ -5,13 +5,14 @@
     text "MyText"
     multiple true
 
-    factory :tasuku_taskables_question_with_options, aliases: [:question_with_options] do
+    factory :tasuku_taskables_question_with_options, aliases: [:question_with_options] do 
```

Comment:
Now I'm just being a dick, but there's whitespace at the end of this line that shouldn't be there.